migreringer/src/main/resources/db/migration/defaultDS/3.1/V3.1_01__TFP-5571-lokalt_oppgave_tabelle.sql

```diff
@@ -0,0 +1,51 @@
+create table OPPGAVE
+(
+    ID NUMBER(19) not null
+        constraint PK_OPPGAVE
+            primary key,
+    TYPE VARCHAR2(20 char) not null,
```

Comment:
Trenger du type her? Fpfordel skal vel bare lage JFR. 
Selv om UI utvides til VurderKonsekvens, Dokument og Henvendelse - så opprettes de i Gosys, som regel manuelt eller av andre system

Reply:
Sant dette - om vi ikke har en plan å bruke det for resten så er det greit å bare fjerne den.